eq-author/src/components/ContentPickerv2/DestinationPicker/Menu.test.js

```diff
@@ -126,131 +134,173 @@ function defaultSetup() {
-      "background-color",
-      colors.lighterGrey
-    );
-
-    // changes with Enter
-    keyPress(tabTitles.later, { key: "Enter", code: "Enter" });
-    expect(getByText(tabTitles.later)).toHaveStyleRule(
-      "background-color",
-      colors.lighterGrey
-    );
-
-    // changes with Space
-    keyPress(tabTitles.other, { key: " ", code: "Space" });
-    expect(getByText(tabTitles.other)).toHaveStyleRule(
-      "background-color",
-      colors.lighterGrey
-    );
-  });
+    it("should display 'Other destinations' options", () => {
+      const { getByText, click } = defaultSetup();
+      click(tabTitles.other);
+      expect(getByText(tabTitles.other)).toHaveStyleRule(
+        "background-color",
+        colors.lighterGrey
+      );
+      expect(getByText(destinationKey[NextPage])).toBeVisible();
+      expect(getByText(destinationKey[EndOfCurrentSection])).toBeVisible();
+      expect(getByText(destinationKey[EndOfQuestionnaire])).toBeVisible();
+    });
 
-  it("should be able to change destination tabs with click", () => {
-    const { getByText, click } = defaultSetup();
+    it("should display only display 'Current section' and 'Other destinations' tab when missing destinations in the 'Later sections'", () => {});
 
-    click(tabTitles.later);
-    expect(getByText(tabTitles.later)).toHaveStyleRule(
-      "background-color",
-      colors.lighterGrey
-    );
-  });
+    it("should be able to change destination tabs with Space or enter", () => {
+      const { getByText, keyPress } = defaultSetup();
 
-  it("should be able to select destination with Enter", () => {
-    const { onSelected, keyPress } = defaultSetup();
+      expect(getByText(tabTitles.current)).toHaveStyleRule(
+        "background-color",
+        colors.lighterGrey
+      );
 
-    keyPress("Question two", { key: "Enter", code: "Enter" });
-    expect(onSelected).toHaveBeenCalledTimes(1);
-    expect(onSelected).toHaveBeenCalledWith(props.data.pages[1]);
-  });
+      // doesn't change with other keyDown
+      keyPress(tabTitles.later, { key: "Escape", code: "Escape" });
+      expect(getByText(tabTitles.current)).toHaveStyleRule(
+        "background-color",
+        colors.lighterGrey
+      );
 
-  it("should be able to select destination with Space", () => {
-    const { onSelected, keyPress } = defaultSetup();
+      // changes with Enter
+      keyPress(tabTitles.later, { key: "Enter", code: "Enter" });
+      expect(getByText(tabTitles.later)).toHaveStyleRule(
+        "background-color",
+        colors.lighterGrey
+      );
 
-    keyPress("Question three", { key: " ", code: "Space" });
-    expect(onSelected).toHaveBeenCalledTimes(1);
-    expect(onSelected).toHaveBeenCalledWith(props.data.pages[2]);
-  });
+      // changes with Space
+      keyPress(tabTitles.other, { key: " ", code: "Space" });
+      expect(getByText(tabTitles.other)).toHaveStyleRule(
+        "background-color",
+        colors.lighterGrey
+      );
```

Comment:
The same as above regarding testing the active indicator rather than the styles.